src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -28,7 +28,7 @@
  *
  * Parameter and service keys are case insensitive.
  *
- * A service id can contain lowercased letters, digits, underscores, and dots.
+ * A service id can contain letters, digits, underscores, and dots.
  * Underscores are used to separate words, and dots to group services
```

Comment:
I removed the whole block =/  This is just somes service id convention, not really related to the container at all; as it allows any character as mentioned.

The part about creating a method is outdated, as it only checks the method map now.